client/src/test/java/io/pravega/client/batch/impl/BatchClientImplTest.java

```diff
@@ -414,7 +417,7 @@ public Void answer(InvocationOnMock invocation) {
         AssertExtensions.assertThrows(SegmentTruncatedException.class, () -> client.getNextStreamCut(startingSC, 50L));
     }
 
-    @Test(timeout = 5000)
+    @Test(timeout = 200000)
```

Comment:
I assume this was for debugging and you meant to change it back.

Reply:
done , now passing number of attempts as 2 for using a constructor visible for testing.